tools/esdlcomp/esdlcomp.h

```diff
@@ -605,6 +605,12 @@ class ParamInfo
         {
             out.appendf("\t\t<EsdlElement name='%s'", name);
             toStringXmlAttr(out);
+
+            if (xsdtype && *xsdtype && hidlMode)
+            {
+                out.appendf(" hidlType='%s'", xsdtype);
```

Comment:
Looking at esxdl I see it uses underscores not camel case.  hidl_type would match other attributes.
But now that I see how this works I think an even better name would be xsd_extended_type.  That's what we are really outputting.  Hidl xsd compatibility really means "using the xsd extended type".